common-messaging/src/main/resources/application.conf

```diff
@@ -15,7 +15,7 @@ messaging {
     password = ${?NATS_PASSWORD}
   }
   kinesis {
-    appName = "ota-plus"
+    appName = ""
```

Comment:
I'll rename it and remove the blank var. I seem to remember needing it to be blank for it to compile, but there should always be one in the app config anyway, so this is redundant.